modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -99,36 +99,22 @@
 			padding-bottom: var( --n-tabs-title-padding-bottom );
 			padding-left: var( --n-tabs-title-padding-left );
 			border-radius: var( --n-tabs-title-border-radius );
-			transition: all var( --n-tabs-title-transition );
 			height: var( --n-tabs-title-height );
 			width: var( --n-tabs-title-width );
 
+			&,
+			span * {
+				transition: all var( --n-tabs-title-transition );
+			}
+
 			&-text {
 				display: flex;
 			  	align-items: center;
 			  	font-size: var( --n-tabs-title-font-size );
 			}
 
-			&:hover {
```

Comment:
I have moved the &:hover styling downwards.

Reply:
I see that `--n-tabs-title-background-color-hover` has no assignment.

Reply:
Can I delete it? Is it managed by the php?
Or should we set the default value?